### ISSUE TITLE
[feature] Server OAPIF simple transactions

### ISSUE THREAD

**Issue by QGIS-feature-tracker** [qgis/QGIS-Documentation#4421]
Original commit: https://github.com/qgis/QGIS/commit/2eb410844fb1b5faf6a1e1f4e2edc96b299f2eba by web-flow

[feature] Server OAPIF simple transactions

**Comment by elpaso** [comment]
I'm not sure about documenting this feature yet, unless we clearly state that the implementation is based on a draft of the specifications and when the specification will be officially published we will update the implementation accordingly, this is to avoid people start building clients that use this API before the specification is finalized.

**Comment by DelazJ** [comment]
@elpaso This is a 3.12 feature so nothing we would reasonably begin to work on before February I think. According to plans 3.10 docs is released when it becomes LTR (hence when 3.12 is released) so maybe should we postpone that to that moment (any idea when the specification is clarified?)

**Comment by elpaso** [comment]
No, sorry. But I'll take care of this when it will be published.

**Comment by DelazJ** [comment]
@elpaso Any suggestion about this feature?
